### ISSUE TITLE
Better visual design

### ISSUE THREAD

**Issue by innerlee** [mkloubert/vscode-kanban#15]
This deserves a better looking UI ;)

**Comment by mkloubert** [comment]
Ideas?

**Comment by samael205** [comment]
I understand that it is possible to modify the CSS at will, so the user believes that, following the guidelines that have been previously established, you can achieve the desired result. ¿No?

**Comment by C0smin** [comment]
> I understand that it is possible to modify the CSS at will, so the user believes that, following the guidelines that have been previously established, you can achieve the desired result. ¿No?

...and maybe share the result? 😁

**Comment by KarlMoellerDev** [comment]
I totally agree. The default UI is especially a strain on the eyes if you are working with a dark vscode theme. For now I have created my own dark theme with some extra adjustments. You can get it [here](https://gist.github.com/KarlMoellerDev/d493cf89ad1ce6b4041a2cf2e31eab1f) if you are interested.
![image](https://user-images.githubusercontent.com/11772241/67849376-ca110780-fb06-11e9-8d93-8f238ec6c100.png)

**Comment by blake-mealey** [comment]
> I totally agree. The default UI is especially a strain on the eyes if you are working with a dark vscode theme. For now I have created my own dark theme with some extra adjustments. You can get it [here](https://gist.github.com/KarlMoellerDev/d493cf89ad1ce6b4041a2cf2e31eab1f) if you are interested.
> ![image](https://user-images.githubusercontent.com/11772241/67849376-ca110780-fb06-11e9-8d93-8f238ec6c100.png)

Hey @FreshlyBrewedCode this link no longer works. Mind making it available elsewhere?
